src/components/earn/PoolCardTri/index.tsx

```diff
@@ -143,14 +118,19 @@ const DefaultPoolCardtri = ({
       currenciesQty={currenciesQty}
       onClick={onCardClick}
     >
+      {stakingInfo && (
+        <ClaimRewardModal
+          isOpen={showClaimRewardModal}
+          onDismiss={() => setShowClaimRewardModal(false)}
+          stakingInfo={stakingInfo}
+        />
+      )}
```

Comment:
`stakingInfo` shouldn't be added to this, bc react only does a shallow comparison of state+props on rerender, this requires a deep comparison to detect changes of nested values

`ClaimRewardModalTri.tsx` only needs `earnedAmount` and `stakedAmount`, which can be passed in from `EarnTri.tsx`

Reply:
Good suggestion! I now refactored PoolCardTri props for avoiding passing StakingInfo as prop in commit 8755ce6